eth/backend.go

```diff
@@ -249,7 +242,7 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 		if _, err := vote.NewVoteManager(
 			eth,
 			chainDb,
-			chainConfig,
+			eth.blockchain.Config(),
```

Comment:
@Francesco4203 it's better u asign an variable 
chainConfig := eth.blockchain.Config(), then pass it for avoiding changing so much ?

Reply:
same with genesisHash

Reply:
Agree, fixed